assets/css/map/controls/_user_location_control.scss

```diff
@@ -0,0 +1,34 @@
+.c-user-location-button {
+  @extend .inherit-box, .border-box;
+
+  color: $color-button-secondary;
+
+  &[disabled] {
+    color: $color-font-grey;
+    background-color: $color-button-disabled;
+  }
+
+  &,
+  &__svg {
+    width: 100%;
+    height: 100%;
+
+    display: flex;
+    align-items: center;
+    justify-content: center;
+  }
+
+  &__svg {
+    svg {
```

Comment:
**suggestion (non-blocking):** not sure how you feel about this, but instead of nesting `svg` inside of `&__svg` you could also just write one selector (`&__svg svg`) given that the outer rule / selector doesn't contain anything aside from the inner one.

Reply:
[`9660272` (#2233)](https://github.com/mbta/skate/pull/2233/commits/96602722816583c57a5510ceb90cd182ec0940e1)